iOS/UI/Common/TabBarController.swift

```diff
@@ -244,6 +233,8 @@ extension TabBarController {
     }
 
     override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
```

Comment:
not much point calling this since the default impl does nothing https://developer.apple.com/documentation/uikit/uiviewcontroller/viewdidlayoutsubviews()